pulp_rpm/app/migrations/0034_modulemd_static_context.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.20 on 2021-05-06 18:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('rpm', '0033_auto_publish'),
```

Comment:
Note to self that if we need to back out the changes, we need to come back and fix this